addons/pos_coupon/static/src/js/OrderWidget.js

```diff
@@ -3,9 +3,15 @@ odoo.define('pos_coupon.OrderWidget', function (require) {
 
     const OrderWidget = require('point_of_sale.OrderWidget');
     const Registries = require('point_of_sale.Registries');
+    const { onWillRender } = owl;
 
     const PosCouponOrderWidget = (OrderWidget) =>
         class PosCouponOrderWidget extends OrderWidget {
+            setup() {
+                onWillRender(() => {
```

Comment:
The listeners in the super class is not activated because you didn't call `super.setup()` here. That's why selecting a line doesn't work.

Reply:
I totally missed that!thank you @caburj